### ISSUE TITLE
Yolov3 number of layers

### ISSUE THREAD

**Issue by behnamj** [AlexeyAB/darknet#3032]
Hey guys, 

Sorry if I ask this naive question, but according to yolov3 paper, the network has 53 conv layers. But when I went through fine-tuning section, I saw yolov3.conv.81 which 81 represents the layer 81. I am confused. What does this 81 indicates?
The other question is about the fine tuning. Is it possible to arbitrarily freeze any number of layers here?

Thanks,

**Comment by AlexeyAB** [comment]
@behnamj Hi,

1. In yolov3 paper is written that https://github.com/AlexeyAB/darknet/blob/master/cfg/darknet53.cfg contains 53 conv layers, not in yolov3. Yolo v3 has additional layers (darknet53.cfg  + ...)

2. In addition to the convolutional layers, there are other layers.

You can use `stopbackward=1` in the any layer, so will be trained only all layers after this layer.

**Comment by behnamj** [comment]
Thanks a lot. Now, I see the confusion.

**Comment by behnamj** [comment]
The other question is when I search for [convolutional] in yolov3.cfg, I only find 75. So where does this 81 come from? 
In [link](https://github.com/AlexeyAB/darknet/blob/6d44529cf93211c319813c90e0c1adb34426abe5/cfg/yolov3.cfg#L548), the stopbackward is in line 548 which is after 52nd convolutional layer. The number 81 doesn't make sense.

Thanks,

**Comment by behnamj** [comment]
The other question is for fine-tuning, is 0.001 large for learning rate? Do we need to decrease it?
